tests/functional/conversion/test_decimal.py

```diff
@@ -0,0 +1,22 @@
+from decimal import Decimal
+
+import pytest
+
+
+def test_converting_formatted_float_strings_to_decimal(convert):
```

Comment:
nit: I tend to use `-k` to run multiple tests and it works great if following this pattern:

```
test_<method-name>_<rest>`
```

```suggestion
def test_convert_formatted_float_strings_to_decimal(convert):
```

Reply:
I reformatted the test names per your suggestion, and added some more to check other bad strings per @fubuloubu's comment above.